src/components/application_manager/src/application_data_impl.cc

```diff
@@ -582,6 +582,134 @@ void DynamicApplicationDataImpl::set_display_layout(const std::string& layout) {
   window_templates_[DEFAULT_WINDOW] = template_config;
 }
 
+/**
+ * @brief AlignKeyboardLayouts aligns content of
+ * supported_keyboard_layouts param with content of configurable_keys param.
+ * @param keyboard_capabilities Aligned keyboard capabilities
+ */
+void AlignKeyboardLayouts(smart_objects::SmartObject& keyboard_capabilities) {
```

Comment:
This looks like it will likely not be needed if these revisions are accepted, as a note: https://github.com/smartdevicelink/sdl_evolution/pull/1117

Reply:
Changes related to revision are added in a8177d9
All these checks with corresponding UTs are removed from feature branch